core-bundle/src/EventListener/TwoFactorFrontendListener.php

```diff
@@ -64,12 +58,7 @@ public function onKernelRequest(GetResponseEvent $event): void
         $request = $event->getRequest();
         $token = $this->tokenStorage->getToken();
 
-        if (null === $token) {
-            return;
-        }
-
-        // Check if is a supported token
-        if (!$token instanceof TwoFactorToken && !\in_array(\get_class($token), $this->supportedTokens, true)) {
+        if (!$token instanceof TwoFactorToken) {
```

Comment:
What's the reason to change this? Now a logged in user will no longer be forced to enable 2FA if it's enforced in the root page?

Reply:
My fault, did miss this case. It's reverted.